### PR TITLE
New designs

### PR DESCRIPTION
**Pull request by evulins**
This PR mainly includes design changes for the following:
- Modal (on mobile and desktop devices)
- Graph /scrollbar/time range buttons (on mobile and desktop)
- Session List Header (on mobile devices)
- Separate Threshold Configuration from modal
- Add "back to map" button

This is part of this[ trello task](https://trello.com/c/P9tr2PaU/1829-session-details-default-time-periods)
Screenshot 2024-07-03 at 13 19 15
Screenshot 2024-07-03 at 13 19 26
Screenshot 2024-07-03 at 13 18 50

### CODE REVIEW

**Comment by JoannaThelema** [comment]
We've lost threshold configurator for mobile sessions. 
Screenshot 2024-07-05 at 14 23 33

File: app/javascript/react/components/Modals/SessionDetailsModal/SessionInfo.tsx [hunk 1]
```diff
@@ -56,102 +72,165 @@ const SessionInfo: React.FC<SessionInfoProps> = ({ sessionType, streamId }) => {
...
-              />
-              {t("sessionDetailsModal.averageValue")}
-              <S.AverageValue>{averageValue}</S.AverageValue>
-              {streamShortInfo.unitSymbol}
-            </S.AverageValueContainer>
-            <S.MinMaxValueContainer>
-              <div>
-                <S.SmallDot
-                  $color={getColorForValue(thresholds, minMeasurementValue)}
+      <S.MobileHeader>
```
Comment (JoannaThelema) [inline app/javascript/react/components/Modals/SessionDetailsModal/SessionInfo.tsx:75]: WDYT about extracting some components from this file?

**Review by JoannaThelema — COMMENTED** [review]
Looks good to me 🙌 I would just split one component into some smaller parts.

**Review by Sonje — APPROVED** [review]
Looks good to me 🚀

**Pull request merged**